src/System.Data.SqlClient/tests/ManualTests/DataCommon/CheckDatabaseIsPresentFactAttribute.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Data.SqlClient.ManualTesting.Tests
+{
+    public class CheckDatabaseIsPresentFactAttribute : FactAttribute
```

Comment:
The general recommendation in CoreFx is to use `ConditionalFact` 
`CheckConnStrSetupFact` was setup before `ConditionalFact` was introduced.

Reply:
This particular check needs a parameter. You could add a static in the test files which need to check for the IsDatabasePresent(string) , and pass in the Database name in the test specific static method.